OMGVA PoS/Business layer/Controllers/UserController.cs

```diff
@@ -116,6 +134,9 @@ public IActionResult Update([FromBody] UpdateUserRequest user, long id)
 
             try
             {
+                if(user.Email != null && !user.Email.IsValidEmail())
```

Comment:
IsValidEmail checks if the string is null therefore the first check is unnecessary.